volontulo/templates/common/badges.html

```diff
@@ -1,12 +1,11 @@
 {% if badges %}
-    <h2>Odznaki użytkownika</h2>
-    <ul class="badges">
+    <ul class="badges list-unstyled">
         {% for badge in badges %}
             <li>
-                <span class="name priority-{{ badge.badge__priority }}">{{ badge.badge__name }}</span> x <span class="badge">{{ badge.badges }}</span>
+                <b class="name priority-{{ badge.badge__priority }}">{{ badge.badge__name }}</b> <span class="badge">{{ badge.badges }}</span>
```

Comment:
but didn't we agree earlier that `b` will stand for medium weight and `strong' will remain bold? 'D